### DISCUSSION TITLE
Contributing to projects

### DISCUSSION THREAD

**Discussion by ghost** [community/community#67972] [Programming Help]
### Body

Hi,

I am looking for some HTML, CSS and JavaScript  projects to contribute on to put on my Resume.

I have a basic understanding of frontend development.

I am willing to work for free because I just want to do it for the experience.

I have no phone and I am at school from 8am-2:30pm.

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by ricardobar96** [comment]
Hi,
An easy way to contribute in some projects would be to look for open source projects that work with your desired language and start contributing there after following the author instructions.

You can check this out here:
https://github.com/trending/javascript
